### PR TITLE
Added real time to MonitorClient with SignalR

### PR DESCRIPTION
**Pull request by ShodytheWolf**
ClientMonitor now shows correctly the users on the attention place of any office (bound to change); the users on the queue waiting to get on the attention places get automatically added to any available attentionPlace in real time.

# Important
there was also a change on the logic of the hub calls inside Commercial-Office, specifically in OfficeService,cs in the function ReleasePosition(string officeId, long placeNumber).

Basically what it does is to call the Hub function "RefreshMonitor" when there is no one in the queue waiting, so that the ClientMonitor page gets updated when an attention place without anyone on the queue gets updated.

https://github.com/user-attachments/assets/57c9a8af-4145-4159-b773-6f2ef1b68952

### CODE REVIEW

**Comment by BraG29** [comment]
si
